### PR TITLE
Add info about request source and target packages

### PR DESCRIPTION
**Pull request by saraycp**
Add info about request source and target packages and also a sentence about release if there is any.

For now, all of this is displayed in the Overview tab of the redesigned request show page.

**Before redesign:**

Screenshot 2022-08-05 at 14-59-09 Request 2 (review)

**After redesign:**

Screenshot 2022-08-05 at 14-33-48 Open Build Service

### CODE REVIEW

**Comment by hennevogel** [comment]
How about 

183082392-acee1078-5f6d-4c2f-b187-4569059eeeff

**Comment by danidoni** [comment]
> How about
> 
> 183082392-acee1078-5f6d-4c2f-b187-4569059eeeff

I like it! I too wasn't sure about having the source + target below the tabs...

File: src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+%p= request_action_header(action, bs_request.creator)
+- if action[:type] == :maintenance_incident && action[:releaseproject] # Is there some release info?
```
[the remark this answers is not in the document]
Comment (saraycp) [reply src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml]: I got rid of the maintenance part.

**Comment by saraycp** [comment]
> How about
> 
> 183082392-acee1078-5f6d-4c2f-b187-4569059eeeff


I like the idea too. I simplified the title and moved the rest of the information under the title.

**Review by eduardoj — APPROVED** [review]

**Comment by hennevogel** [comment]
codecov reporting hangs. [coverage job is fine](https://app.circleci.com/pipelines/github/saraycp/open-build-service/1769/workflows/e92a165a-ee3c-4971-9c33-6e865ad375d3/jobs/11855). merging!

**Pull request merged**